Lab4

WyattOtsuka opened the pull request:
@MinniVan Ready for the review! Sorry I'm so late on it, I had to do an unholy amount of rebasing, merge conflict solving, and commit amending to try to remove the h5 file, but here it is!

WyattOtsuka committed:
problem 1 and part of problem 2

WyattOtsuka committed:
started p2

WyattOtsuka committed:
finished p2, fixed p1

WyattOtsuka committed:
finished problem #3

WyattOtsuka committed:
added html version

WyattOtsuka committed:
implemented feedback

WyattOtsuka committed:
setting up branch

WyattOtsuka committed:
started problem 1, graphed long. vs count

WyattOtsuka committed:
Adjusting graphs for 1

WyattOtsuka committed:
adjusted exponential overlay on problem 1

WyattOtsuka committed:
adjusting graphs for 1.2 and 1.3, explaining 1.3

WyattOtsuka committed:
finished 1.3, started 1.4

WyattOtsuka committed:
finished problem 1

WyattOtsuka committed:
finished problem 2

WyattOtsuka committed:
ran file for better pull request

WyattOtsuka committed:
ignoring h5 files

WyattOtsuka committed:
removed large files

MinniVan reviewed (commented):
The notebook is too large to open here and make comments. I've sent you an email with an annotation of your file.

WyattOtsuka committed:
added partner feedback
Changed from 270 to 315 in jump location for problem 1, adjusted mean vs longitude graph for more clarity, added semilog plot

WyattOtsuka committed:
added html version

WyattOtsuka commented:
@hrzhao76 Ready for review

hrzhao76 commented:
Hi, just a follow-up of our discussion during the lab.

I reviewed your code. In my opinion the way you calculate mean and std also includes the faint stars, so it is not very accurate. You can try to draw your model and compare with distribution of one image and see if your mean and std successfully fit the shape of the histogram of the image.

There are other ways to better estimate the bkg. e.g, if you draw the distribution of each image overall they seem similar. But doing some operation will cause the shape of bkg separate from faint stars of transient.  what if you add two images? what about subtracting one from another? 

Think about this :)
